lang/python/python-uci/Makefile

```
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2018-2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-uci
+PKG_VERSION:=0.8.1
+PKG_RELEASE:=2
```

This package was not present in OpenWrt packages feed before, so we should start with 1, but there was a recent change that there is a possibility to use autorelease.

```suggestion
PKG_RELEASE:=$(AUTORELEASE)
```